scripts/powershell/force-push-to-gh-pages.ps1

```diff
@@ -0,0 +1,38 @@
+# ASSUMPTIONS BEFORE RUNNING
```

Comment:
Where does this get used? Should it be under eng/common?

Reply:
Its used for releasing to github.io. Right now we just run it off DevOps releases, it has no yml definition. [Here](https://dev.azure.com/azure-sdk/internal/_release?_a=releases&view=all&path=%5CGitHub)

I don't think it needs to be synced across repos

Reply:
OK did you plan to update those release pipelines to use this from the new location? 

As a follow-up to your shared tools epic can you add an issue about converting our release pipelines into yml based unified pipelines so that we can move those to a repo, hopefully our public repo.